File: Beecrowd/Level1/bee-1020.py
age = int(input())

years = int(age / 365)
months = int((age - years * 365) / 30)
days = int(age - (years * 365) - (months * 30))

print(f'{years} ano(s)')
print(f'{months} mes(es)')
print(f'{days} dia(s)')

# 1020 - Age in Days
# https://www.beecrowd.com.br/judge/en/problems/view/1020

File: C/Beecrowd/bee-1020.c
#include <stdio.h>

int main(){
    int days, years, months;
    scanf("%d", &days);

    years = days/365;
    days %= 365;
    months = days/30;
    days %= 30;

    printf("%d ano(s)\n%d mes(es)\n%d dia(s)\n", years, months, days);

    return 0;
}

// 1020 - Age in Days
// https://www.beecrowd.com.br/judge/en/problems/view/1020